screwdriver/screwdriver.go

```diff
@@ -96,7 +96,13 @@ type Validator struct {
 // Pipeline is a Screwdriver Pipeline definition.
 type Pipeline struct {
 	ID     string `json:"id"`
-	ScmURL string `json:"scmUrl"`
+	ScmRepo ScmRepo `json:"scmRepo"`
+	ScmUri string `json:"scmUri"`
+}
+
+// ScmRepo is an object inside a Pipeline
```

Comment:
Fixed!
The new API will have a `scmRepo` object inside the pipeline.
https://beta.api.screwdriver.cd/v4/pipelines